README.md

```diff
@@ -277,6 +277,7 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
 * [ngrading](https://github.com/MikuAuahDark/NPad93/tree/master/ngrading) - Simple color grading library.
 * [Shadertoy viewer](http://love2d.org/forums/viewtopic.php?f=5&t=80885) - Run code copied from shadertoy directly or output the converted code to a LÖVE shader.
 * [Moonshine](https://github.com/vrld/moonshine) - Repository of common post-processing effects like blur, vignette, color-grading, etc.
+* [ShaderScan](https://github.com/idbrii/love-shaderscan) - better iteration with shaders: hot reload, includes, error messages.
```

Comment:
```suggestion
* [ShaderScan](https://github.com/idbrii/love-shaderscan) - better iteration with shaders, such as hot reload, includes, and error messages.
```

Reply:
Rephrased differently to put the features first, but still eliminate the colon.